src/components/MessageBar/message-bar.css

```diff
@@ -0,0 +1,36 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+@import "src/css/variables/fonts.css";
+
+.y-message-bar {
+  display: flex;
+  border-radius: 1px;
+  border-left-width: 4px;
+  border-left-style: solid;
+  background-color: $color__popRock;
+  padding: 7px 12px 8px 12px;
+  font-size: $fontSize__small;
+  line-height: $lineHeight__small;
+}
+
+.y-message-bar--message {
+  flex-grow: 1;
+  color: $color__blackMetal;
+}
+
+.y-message-bar--actions {
+  padding-left: 32px;
+  flex-grow: 0;
+}
+
+.y-message-bar__type-info {
+  border-left-color: $color__punkRock;
+}
+
+.y-message-bar__type-warning {
+  border-left-color: $color__noon;
+}
+
+.y-message-bar__type-error {
+  border-left-color: $color__angel;
```

Comment:
You could alias these in the `colors.css` file, and have something like `$messageBarColor__error` in this file.

Reply:
mmm I like the locality of putting the message color in this css file. Do we really want styling info for this component in colors.css?

Reply:
It's not the styling info. It's providing semantic names to colors. All component frameworks do this: Fabric, Bootstrap, Foundation, etc. Nobody litters their component CSS files with random color names, but with semantic names.

Reply:
See for example https://github.com/twbs/bootstrap/blob/9a1d81f154f4391f9fea47fc26cd2fa864d6ae8a/scss/_buttons.scss#L81-L82

They create names for the colors, and then use the names.